docs/delete_guest_credential.md

```diff
@@ -0,0 +1,35 @@
+# delete_guest_credential
+
+Delete a guest credential from the Rubrik cluster.
+```py
+def delete_guest_credential(username, domain=None, timeout=15):
+```
+
+## Arguments
+| Name        | Type | Description                                                                 | Choices |
+|-------------|------|-----------------------------------------------------------------------------|---------|
+| username | str  | The account username to be deleted.          |         |         |
+
+## Keyword Arguments
+| Name        | Type | Description                                                                 | Choices | Default |
+|-------------|------|-----------------------------------------------------------------------------|---------|---------|
+| domain  | int  | The domain name of the account to be deleted. |         |
+| timeout  | int  | The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error.  |         |    15     |
+
+## Returns
+| Type | Return Value                                                                                   |
+|------|-----------------------------------------------------------------------------------------------|
+| str  | No change required. The guest credential 'username' does not exist. |
+| str  | No change required. The guest credential 'username@domain' does not exist. |
+| dict  | The full API response for `/vmware/guest_credential`. |
```

Comment:
What type of call is happening for `/vmware/guest_credential`? (Guessing its DELETE but lets be sure to call that out)

Reply:
Also, which version of the API?